packages/components/src/components/calendar/calendar.tsx

```diff
@@ -52,7 +92,7 @@ const Calendar = React.memo(
                 }
             };
 
-            const onMouseOver = event => {
+            const onMouseOver = (event: React.MouseEvent<HTMLSpanElement>) => {
```

Comment:
just for clarification: is it really suitable to apply in this file HTMLSpanElement for these events?

Reply:
Yes @yauheni-deriv, because as seen in the attached screenshot, these events are associated with `span` elements. It's better to be explicit in my opinion. Why do you suggest I remove it?

<img width="1129" alt="event" src="https://user-images.githubusercontent.com/104334373/214500580-2e9f7ab9-4357-4fa8-86fb-328e528e6d7e.png">

Reply:
No, I didn’t mean to remove it. thanks for clarification, I just missed the type of the element